### PR TITLE
Kurtwheeler/api tests no internet

### PR DESCRIPTION
**Pull request by kurtwheeler**
## Issue Number

#1878 

## Purpose/Implementation Notes

This PR makes it so the API tests don't hit external APIs so that our tests' success isn't dependent on them being up. This is the first of a few PRs to get us to the point where we don't hit external APIs from our test suite.

The API sub-project was easy because there were just a few places where the API made external calls: when initializing organism models for the tests. I just changed the way we initialized these to not required hitting NCBI's API and it worked!

## Types of changes

- New feature (non-breaking change which adds functionality)

## Functional tests

I turned off my internet and ran the API tests.

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by arielsvn** [comment]
I like this way of creating organisms! Is it possible to abstract it in some way? Maybe have a test helper file somewhere with a `create_organism` function?

```
ailuropoda = Organism(name="AILUROPODA_MELANOLEUCA", taxonomy_id=9646, is_scientific_name=True)
ailuropoda.save()
```

Looks like we are repeating this logic in multiple places.

**Comment by davidsmejia** [comment]
🔥 🔥 🔥

File: scripts/common.sh [hunk 1]
```diff
@@ -8,7 +8,12 @@ export CCDL_WORKER_IMAGES="smasher compendia illumina affymetrix salmon transcri
...
     if [ "$(uname)" = "Linux" ]; then
-        ip route get 8.8.8.8 | grep -oE 'src ([0-9]{1,3}\.){3}[0-9]{1,3}' | awk '{print $2; exit}'
+        # If we're not connected to the internet return an error code.
+        ip_out=$(ip route get 8.8.8.8)
+        if [ $? -ne 0 ]; then
+            return 1
+        fi
+        echo $ip_out | grep -oE 'src ([0-9]{1,3}\.){3}[0-9]{1,3}' | awk '{print $2; exit}'
     elif [ "$(uname)" = 'Darwin' ]; then # MacOS
         ifconfig | grep "inet " | grep -v 127.0.0.1 | cut -d\  -f2 | tail -1
```
Comment (davidsmejia) [inline scripts/common.sh]:
```suggestion
         ip_out=$(ifconfig | grep "inet " | grep -v 127.0.0.1 | cut -d\  -f2 | tail -1)
         if [ -z "$ip_out" ]; then
                 return 1
         fi
         echo $ip_out 

```

Comment (davidsmejia) [reply scripts/common.sh]: Might want to have this also work on 🍎

**Comment by kurtwheeler** [comment]
@arielsvn I'm not sure about that one. This is creation of test data, which is something we tend to repeat a lot... It's also not clear what the function would do for taxonomy ids. If you feel strongly about this one we can discuss it more but I feel like I'm generally okay with keeping test code a bit DAMP.

**Comment by arielsvn** [comment]
Yeah let's talk it, I'm not talking about removing repetition but making it more descriptive. I would prefer if we could have `test_utils.create_organism()`, instead of the whole to logic create the organism. From the tests it looks like we only care about the organism, the actual name and taxonomy id are important?

**Pull request merged**